packages/atomic/src/index.html

```diff
@@ -107,7 +107,7 @@
         margin: 10px 0;
       }
 
-      @media only screen and (max-width: 1024px) {
+      @media only screen and (max-width: 1023px) {
```

Comment:
at the 1024px width 
<img width="1011" alt="Screen Shot 2021-04-30 at 1 25 38 PM" src="https://user-images.githubusercontent.com/4923043/116739577-d5761500-a9c1-11eb-9202-f8d668ed88c7.png">

Reply:
Why does reducing `max-width` by 1 pixel solve the issue? And what would happen if there were three facets instead of four?

Reply:
because tailwind is going the other way, meaning it uses min-width 1024px to apply rules for bigger screen.
Another solution would be to do the same in index.html, build mobile first & then apply desktop rules.

Reply:
Problem is independent of the number of facets, they will still appear on one line instead of beside